Makefile

```diff
@@ -30,6 +30,9 @@ SAIL_DEFAULT_INST += riscv_insts_zbc.sail
 SAIL_DEFAULT_INST += riscv_insts_zbs.sail
 
 SAIL_DEFAULT_INST += riscv_insts_zfh.sail
+
+SAIL_DEFAULT_INST += riscv_insts_zcb.sail
```

Comment:
It would be good to keep this list sorted alphabetically, i.e. move it up one.

Reply:
This has been updated.